scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
+
+[Service]
+User=haas_user
+Group=haas_user
+WorkingDirectory=/var/lib/haas/
+ExecStart=/usr/bin/haas serve_networks
```

Comment:
So, I don't have super strong feelings about this, but I'd say if you're explicitly setting something you should at least put a comment as to why. `Type=simple` is probably obvious enough to be an exception to this.